client/additional-methods-setup/index.js

```diff
@@ -10,6 +10,8 @@ import { addFilter } from '@wordpress/hooks';
  * Internal dependencies
  */
 import MethodSelector from './methods-selector';
+import UpePreviewMethodSelector from './upe-preview-methods-selector';
+import WcPayUpeContextProvider from '../settings/wcpay-upe-toggle/provider';
```

Comment:
Yeah, I had the same feelings 😄 
I went with the camel case approach out of habit, since I didn't also see a linting rule preventing me from doing it or a clear convention 🤷